### PR TITLE
docs(python): Add docstring example for DataFrame.limit()

### PR DESCRIPTION
**Pull request by montanarograziano**
Related to: Add missing docstring examples #13161

Add example for:
https://docs.pola.rs/py-polars/html/reference/dataframe/api/polars.DataFrame.limit.html

### CODE REVIEW

File: py-polars/polars/dataframe/frame.py [hunk 1]
```diff
@@ -5066,6 +5066,29 @@ def limit(self, n: int = 5) -> Self:
         See Also
         --------
         head
+
+        Examples
+        --------
+        Get the first 3 rows of a DataFrame.
+        >>> df = pl.DataFrame(
```
Comment (stinodego) [inline py-polars/polars/dataframe/frame.py:5073]: This needs a blank line or it won't render correctly:

```suggestion
        Get the first 3 rows of a DataFrame.
        
        >>> df = pl.DataFrame(
```

File: py-polars/polars/dataframe/frame.py [hunk 2]
```diff
@@ -5066,6 +5066,29 @@ def limit(self, n: int = 5) -> Self:
...
+        ┌─────┬─────┬─────┐
+        │ foo ┆ bar ┆ ham │
+        │ --- ┆ --- ┆ --- │
+        │ i64 ┆ i64 ┆ str │
+        ╞═════╪═════╪═════╡
+        │ 1   ┆ 6   ┆ a   │
+        │ 2   ┆ 7   ┆ b   │
+        │ 3   ┆ 8   ┆ c   │
+        └─────┴─────┴─────┘
+
```
Comment (stinodego) [inline py-polars/polars/dataframe/frame.py:5091]: There should be no blank line at the end:
```suggestion
```

**Review by stinodego — CHANGES_REQUESTED** [review]
Thanks - some minor comments on the formatting and this can be merged.

**Comment by montanarograziano** [comment]
Thank you! I've applied the changes you mentioned.

**Review by stinodego — APPROVED** [review]
Great, thanks!

**Pull request merged**